mips/stalin-sort.s

```diff
@@ -0,0 +1,118 @@
+	.data
+	
+unsortedArray:
+	.word 1, 2, 4, 3, 6, 8, 0, 9, 5, 7
```

Comment:
what happens with repeated numbers like `.word 1, 2, 4, 3, 4, 8, 0, 9, 5, 7`?

Reply:
It keeps the repeats as shown below. Check line 64 where branches to skip adding the unsorted element only if it is less than the most recently added element to the sorted list.

![image](https://user-images.githubusercontent.com/7423376/63352029-55fbfb80-c32e-11e9-8de7-00eaae54591c.png)

Reply:
it's been ages since I saw a MIPS assembly code. great work!